src/test/java/com/aws/greengrass/builtin/services/pubsub/PubSubIPCEventStreamAgentTest.java

```diff
@@ -303,6 +305,15 @@ void GIVEN_subscribed_consumer_WHEN_publish_binary_message_THEN_publishes_And_TH
         assertEquals(0, pubSubIPCEventStreamAgent.getListeners().size());
     }
 
+    @Test
```

Comment:
Let's get a test for the wildcard stuff.

Reply:
added